libs/shared/ui/src/lib/textField/textField.module.scss

```diff
@@ -88,6 +88,10 @@
     border-color: #ffdad4 !important;
 }
 
+.readOnly{
```

Comment:
use this pseudo-class:
https://developer.mozilla.org/en-US/docs/Web/CSS/:read-only